sktime/forecasting/base/adapters/_darts.py

```diff
@@ -27,45 +27,43 @@ class _DartsAdapter(BaseForecaster):
     """
 
     _tags = {
+        # packaging info
+        # --------------
+        "authors": ["yarnabrina", "fnhirwa"],
+        "maintainers": ["yarnabrina", "fnhirwa"],
+        "python_version": ">=3.8",
+        "python_dependencies": ["u8darts"],
+        "python_dependencies_alias": {"u8darts": "darts"},
+        # estimator type
+        # --------------
         "y_inner_mtype": "pd.DataFrame",
         "X_inner_mtype": "pd.DataFrame",
         "requires-fh-in-fit": False,
         "enforce_index_type": pd.DatetimeIndex,
         "handles-missing-data": False,
         "capability:insample": False,
-        "python_version": ">=3.8",
-        "python_dependencies": ["u8darts"],
-        "python_dependencies_alias": {"u8darts": "darts"},
     }
 
     def __init__(
         self: "_DartsAdapter",
         past_covariates: Optional[List[str]] = None,
         num_samples: Optional[int] = 1000,
     ) -> None:
-        super().__init__()
-        if past_covariates is not None and not isinstance(past_covariates, list):
+        if not isinstance(past_covariates, list) and past_covariates is not None:
             raise TypeError(
                 f"Expected past_covariates to be a list, found {type(past_covariates)}."
             )
-        self._past_covariates = [] if past_covariates is None else past_covariates
+        self.past_covariates = [] if past_covariates is None else past_covariates
```

Comment:
This should not be done. `self.<init argument>` must match the value passed in `__init__` exactly, without any change.

Reply:
sure let me add it back